### PR TITLE
feat: Change event type used for accountability

### PR DESCRIPTION
**Pull request by Matovidlo**
Jira: [PSGO-870](https://keboola.atlassian.net/browse/PSGO-870)

**Changes:**
- Change event schema name, ID and made adjustments accordingly based on new event schema.
- Add new components.
- Add source create event.

-----------

[PSGO-870]: https://keboola.atlassian.net/browse/PSGO-870?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: internal/pkg/service/stream/sink/type/tablesink/keboola/bridge/event.go [hunk 1]
```diff
@@ -16,14 +16,15 @@ import (
 	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
 )
 
-// Schema: https://github.com/keboola/event-schema/blob/main/schema/ext.keboola.keboola-buffer.json
-const componentID = keboola.ComponentID("keboola.keboola-buffer")
+// Schema: https://github.com/keboola/event-schema/blob/main/schema/ext.keboola.keboola-stream..json
```
Comment (hosekpeter) [inline internal/pkg/service/stream/sink/type/tablesink/keboola/bridge/event.go:19]:
```suggestion
// Schema: https://github.com/keboola/event-schema/blob/main/schema/ext.keboola.keboola-stream.json
```

**Review by hosekpeter — APPROVED** [review]

File: internal/pkg/service/stream/sink/type/tablesink/keboola/bridge/event.go (hunk 1 above)
Comment (jachym-tousek-keboola) [reply internal/pkg/service/stream/sink/type/tablesink/keboola/bridge/event.go:19]: No, the two dots are intentional.

**Review by jachym-tousek-keboola — APPROVED** [review]

File: internal/pkg/service/stream/sink/type/tablesink/keboola/bridge/event.go (hunk 1 above)
Comment (Matovidlo) [reply internal/pkg/service/stream/sink/type/tablesink/keboola/bridge/event.go:19]: True, I though at first it is mistake, but it is correct. Here is readme https://github.com/keboola/event-schema/blob/main/README.md

Comment (hosekpeter) [reply internal/pkg/service/stream/sink/type/tablesink/keboola/bridge/event.go:19]: Thank you 👍

**Comment by Matovidlo** [comment]
Just leaving comment, that we will merge first 1 PR in event schema and another one in connection to support schema in `event` endpoint. Therefore this will be not merged yet

**Review by jachym-tousek-keboola — APPROVED** [review]

**Pull request merged**